pkg/api/v1beta1/dynakube/dynakube_conditions.go

```diff
@@ -0,0 +1,99 @@
+package dynakube
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// TokenConditionType identifies the token validity condition.
+	TokenConditionType string = "Tokens"
+
+	// APITokenConditionType identifies the API Token validity condition.
+	APITokenConditionType string = "APIToken"
+
+	// PaaSTokenConditionType identifies the PaaS Token validity condition.
+	PaaSTokenConditionType string = "PaaSToken"
+
+	// DataIngestTokenConditionType identifies the DataIngest Token validity condition.
+	DataIngestTokenConditionType string = "DataIngestToken"
+)
+
+// Possible reasons for ApiToken and PaaSToken conditions.
+const (
+	// ReasonTokenReady is set when a token has passed verifications.
+	ReasonTokenReady string = "TokenReady"
+
+	// ReasonTokenError is set when an unknown error has been found when verifying the token.
+	ReasonTokenError string = "TokenError"
+
+	ReasonCreated         string = "Created"
+	ReasonError           string = "Error"
+	ReasonUnexpectedError string = "UnexpectedError"
+	ReasonUpToDate        string = "UpToDate"
```

Comment:
You kinda went against the current pattern 😅 

I would like to avoid bloating the Dynakube package further.

There was this TODO here even related to the TokenScope conditions.
```
// TODO: Move these conditions related consts to a place where they are used, so we don't bloat this package further.
```

Look at https://github.com/Dynatrace/dynatrace-operator/blob/37afc6130005c92ba0ff4fddb503292be48269aa/pkg/util/conditions/error.go#L1-L1 for the common conditions.

Reply:
Also, I would avoid touching the `Token` related conditions as of right now, it is out of scope of your change.

Reply:
Ups, read that we don't want to bloat the file instead of the package. - Will move it to a package more related to activegate, as you described below.

Reply:
Moved it 👍